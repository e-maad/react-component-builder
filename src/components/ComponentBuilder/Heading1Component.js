import React from 'react';

const Heading1Component = ({ text, id }) => {
    return <h1 id={id} className='heading1-component'>{text}</h1>
}

export default Heading1Component;