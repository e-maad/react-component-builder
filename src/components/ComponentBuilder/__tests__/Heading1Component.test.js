import React from 'react';
import Heading1Component from '../Heading1Component';


const wrapper = shallow(<Heading1Component />);
describe('Heading1 Component', () => {

  it('match the snapshot', () => {
    const wrapper = renderer.create(<Heading1Component />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });

  it('base element should be present', () => {
    expect(wrapper.find('.heading1-component').length).toBe(1);
  });
});