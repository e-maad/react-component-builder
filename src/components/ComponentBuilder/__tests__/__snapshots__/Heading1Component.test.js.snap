// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading1 Component match the snapshot 1`] = `
<h1
  className="heading1-component"
/>
`;
